### PR TITLE
fix: presets styling for web

### PR DESCRIPTION
**Pull request by chmelevskij**
* fix presets styling

### CODE REVIEW

**Comment by nerdCopter** [comment]
`master`


`5b7ef3ef`

**Review by nerdCopter — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Pull request merged**
